flow/connectors/clickhouse/cdc.go

```diff
@@ -122,8 +119,16 @@ func (c *ClickHouseConnector) SyncRecords(ctx context.Context, req *model.SyncRe
 	return res, nil
 }
 
+// For CH specifically, we don't want to replay during sync, but rather during the normalization phase.
+// This a no-op for CH, private variant to be called during normalize.
 func (c *ClickHouseConnector) ReplayTableSchemaDeltas(ctx context.Context, flowJobName string,
```

Comment:
should move this method to a separate connector interface, there's a few connectors that skip this already